linera-execution/solidity/linera.sol

```diff
@@ -150,11 +446,20 @@ library Linera {
 
     function message_is_bouncing() internal returns (MessageIsBouncing) {
         address precompile = address(0x0b);
-        bytes memory input1 = new bytes(1);
-        input1[0] = bytes1(uint8(4));
+        bytes memory input1 = new bytes(2);
+        input1[0] = bytes1(uint8(1));
+        input1[1] = bytes1(uint8(4));
```

Comment:
Is this code generated or hand-written? Where did you get the bytes from? Also, it seems like previous version was incorrect? Or why did it change? I don't see any test being updated that would help understanding.

Reply:
As far as I know, the previous version was correct.
The matching changes are in `revm.rs`. The `PrecompileTag` has been reorganized in order to handle those various cases separately and more cleanly than before.

In one way or another we have two codes: The rust code and the solidity code. It is not going to be super nice. So, we need a translation layer and that translation layer is `linera.sol`.